trappy/plotter/ILinePlot.py

```diff
@@ -149,9 +149,17 @@ def __init__(self, traces, templates=None, **kwargs):
     def savefig(self, *args, **kwargs):
         raise NotImplementedError("Not Available for ILinePlot")
 
-    def view(self, test=False):
+    def view(self, max_datapoints=75000, test=False):
         """Displays the graph
 
+        :param max_datapoints: Maximum number of datapoints to plot.
+        Dygraph can make the browser unresponsive if it tries to plot
+        too many datapoints.  Chrome chokes at around 75000, Firefox
```

Comment:
These numbers could vary, Can we mention the specs of the machine this was tested on as well? So if the user has a beast of a machine, they can calibrate accordingly (although I fear the relationship might be non-linear)